podcasts/Onboarding/Models/PlusPricingInfoModel.swift

```diff
@@ -91,6 +95,15 @@ class PlusPricingInfoModel: ObservableObject {
             }
         }
 
+        var terms: String {
+            switch type {
+            case .freeTrial:
+                return L10n.pricingTermsAfterTrialLong(duration, dateAfterOffer )
+            case .discount:
+                return L10n.pricingTermsAfterDiscount(rawPrice, duration, dateAfterOffer )
```

Comment:
If we switch to using the `.long` format, we can use the nonBreakingSpaces() to prevent it from being split across multiple lines. 

```suggestion
                return L10n.pricingTermsAfterTrialLong(duration, dateAfterOffer.nonBreakingSpaces())
            case .discount:
                return L10n.pricingTermsAfterDiscount(rawPrice, duration, dateAfterOffer.nonBreakingSpaces())
```